### PR TITLE
New analytics

### PR DESCRIPTION
**Pull request by csacsi**

### CODE REVIEW

File: Example/Example/ViewControllers/WidgetsExampleViewController.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.row == 51, let widgetCell = tableView.dequeueReusableCell(withIdentifier: "widgetCell") {
+            return widgetCell
+        }
+
+        let cell = UITableViewCell(style: .default, reuseIdentifier: "defaultCell")
+        if indexPath.row == 0 {
+            cell.textLabel?.text = "This example demonstartes when you have to fire 'Opened Widget' and 'Widget Visible'\n\n Be aware of the difference betwen 'Opened Widget' and 'Widget Visible'\n\n 'Opened Widget': You should fire this when firing the api is done and loading the photo data into your own view for the widget is complete.\n\n'Widget Visible': 'Opened Widget' should be fired first. Then, you can fire this when your own view for the widget started to be visible on the screen.\n\nPlease scroll down to see it in action."
```
Comment (jchen123) [inline Example/Example/ViewControllers/WidgetsExampleViewController.swift]: spelling of "demonstrates"

File: PixleeSDK.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -164,6 +165,7 @@
 		F2832B0F23F3E04B004B62C0 /* PXLAnalyticsEvent.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = PXLAnalyticsEvent.swift; sourceTree = "<group>"; };
 		F2832B1123F3E062004B62C0 /* PXLAnalyticsService.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = PXLAnalyticsService.swift; sourceTree = "<group>"; };
 		F2832B1323F3E19F004B62C0 /* PXLPlainErrorDTO.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PXLPlainErrorDTO.swift; sourceTree = "<group>"; };
+		F296941D24223D77003EB9DD /* PXLAnalyitcsEventWidgetVisible.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PXLAnalyitcsEventWidgetVisible.swift; sourceTree = "<group>"; };
```
Comment (jchen123) [inline PixleeSDK.xcodeproj/project.pbxproj]: can we change all spelling of PXLAnalyitcs to PXLAnalytics across the entire repository?

**Comment by SungjunApp** [comment]
Great work for the sample app. **Widgets example** demo works great! WidgetVisible and WidgetOpened are fired in the right situations.

In Analytics, there are a few problems I found.

## 1. These buttons do not respond to the click.
- Opened lightbox
- Action Clicked

## 2. Add to cart and Conversion buttons are overlapped on iPhone SE.
- Left: iPhone SE
- Right: iPhone 11 Pro Max, Simulator

File: PixleeSDK.xcodeproj/project.pbxproj (hunk 2 above)
Comment (csacsi) [reply PixleeSDK.xcodeproj/project.pbxproj]: fixed all over the app

File: Example/Example/ViewControllers/WidgetsExampleViewController.swift (hunk 1 above)
Comment (csacsi) [reply Example/Example/ViewControllers/WidgetsExampleViewController.swift]: fixed

**Comment by csacsi** [comment]
@SungjunApp fixed the layout issue, and added more description for the not responding buttons

File: PixleeSDK.xcodeproj/project.pbxproj (hunk 2 above)
Comment (jchen123) [reply PixleeSDK.xcodeproj/project.pbxproj]: not this file though i see, but is it just because it's not useful?

Comment (csacsi) [reply PixleeSDK.xcodeproj/project.pbxproj]: jeah this is the project of the SDK, and i fixed the spelling problem in the Example app. Now added the fix for here too.

**Comment by SungjunApp** [comment]
Looks good! The sample app works great!

**Pull request merged**